app/views/tasks/_rating_modal.html.slim

```diff
@@ -1,27 +1,17 @@
-.modal.fade#ratingModal aria-hidden="true" data-task-id=@task.id tabindex="-1"
-  .modal-dialog.modal-dialog-centered.modal-sm
-    .modal-content
-      .modal-header
-        h6.modal-title
-          = Rating.model_name.human
-        button.btn-close data-bs-dismiss="modal" aria-label="Close"
-      .modal-body
-        - Rating::SUBCATEGORIES.each do |category|
-          span.rating-category-container
-            span
-              = t("activerecord.attributes.rating.#{category}")
-            span.task-rating
-              = render('task_star_rating', rating: @user_rating.send(category), is_input: true, category:)
-        hr
-        span.rating-category-container
-          span
-            = t('activerecord.attributes.rating.overall_rating')
-          span.task-rating
-            = render('task_star_rating', rating: @user_rating.overall_rating, is_input: true, category: :overall_rating)
-      .modal-footer.pt-0.border-0
-        - if @user_rating.valid?
-          button.btn.btn-light#ratingModalSaveButton data-bs-dismiss="modal"
-            = t('common.button.save')
-        - else
-          button.btn.btn-light#ratingModalSaveButton[data-bs-dismiss="modal" disabled]
-            = t('common.button.save')
+= render 'shared/modal', title: Rating.model_name.human, root_attributes: {id: 'ratingModal', 'data-task-id': @task.id}, size: 'modal-sm', footer: false do
+  .modal-body
+    - Rating::SUBCATEGORIES.each do |category|
+      span.rating-category-container
+        span
+          = t("activerecord.attributes.rating.#{category}")
+        span.task-rating
+          = render('task_star_rating', rating: @user_rating.send(category), is_input: true, category:)
+    hr
+    span.rating-category-container
+      span
+        = t('activerecord.attributes.rating.overall_rating')
+      span.task-rating
+        = render('task_star_rating', rating: @user_rating.overall_rating, is_input: true, category: :overall_rating)
+
+    button.btn.btn-light.float-end.mt-3#ratingModalSaveButton data-bs-dismiss="modal" disabled=@user_rating.invalid?
+      = t('common.button.save')
```

Comment:
Let's move this part to the footer, I'd say (we don't need a close button anyway).